development/extensions/modification_to_extension.rst

```diff
@@ -248,20 +247,14 @@ In this example my **controller/main.php** would look like the following:
 
 .. note::
 
-    The consecution of arguments in services.yml should match the consecution of
+    The order of arguments in services.yml should match the order of
     arguments passed to the class constructor ``public function __construct()``.
-    Otherwise, error will be thrown and the board will be broken if you try to
+    Otherwise, an error will be thrown and the board will be broken if you try to
```

Comment:
Just on a side note on something we might want to say after merging this into 3.2.x:
I think we implemented a feature in phpBB 3.2 that will prevent this from happening, It will however not load the extensions and enter a state of safe mode IIRC.